helm_charts_updater/helm.py

```
@@ -1,48 +1,98 @@
+            A Chart model instance populated from the Chart.yaml file.
+
+        Raises:
+            ChartValidationError: If the Chart.yaml file fails validation.
+        """
+        logging.info("Parsing %s's Chart.yaml...", self.chart_name)
         yaml = YAML(typ="rt")
 
-        with open(f"{self.clone_path}/{self.charts_path}/{self.chart_name}/Chart.yaml", "r") as f:
-            chart = f.read()
+        chart_path = self._get_chart_path()
+        with open(chart_path, "r", encoding="utf-8") as f:
```

Ruff (0.15.7)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)